staging/src/k8s.io/apiserver/pkg/endpoints/handlers/watch.go

```diff
@@ -269,6 +287,15 @@ func (s *WatchServer) HandleWS(ws *websocket.Conn) {
 			}
 			obj := event.Object
 			s.Fixup(obj)
+			if s.ExportOptions.Export && s.Exporter != nil {
+				var err error
+				// TODO: Passing nil for context here as there is no request from which to create one:
+				// ctx *may* be completely unused in all exporters?
+				obj, err = s.Exporter.Export(nil, event.Object, s.ExportOptions)
```

Comment:
Anyone have thoughts on this situation? I'd like to apply export logic as we do up in ServeHTTP or over in the rest.go endpoint handlers, but we have no such request scope here.

Most exporters don't seem to use context, but for example the certificates strategy passes through to this which does: https://github.com/dgoodwin/kubernetes/blob/2568a92119833dcb76628fcf8d3a46e37abc06fe/pkg/registry/certificates/certificates/strategy.go#L66-L66